### DISCUSSION TITLE
Is it possible to integrate LNURL into Phoenix?

### DISCUSSION THREAD

**Discussion by leandro21b** [ACINQ/phoenix#385] [Improving Phoenix]
It would be interesting to have a fixed `LNURL` to receive money without having to create an invoice for each payment, of course as it is a self-custody wallet it would be necessary to leave it running in the background, spending more energy, so the user can have the option of enabling or disable, but would it be possible to integrate in the future?

**Comment by dpad85** [comment]
For now we are focusing on splicing but we are planning to support that with lightning address and BOLT12. As you mention, one of the issues is that the app is not always started, and the device itself is not always online, so reliability will not be 100%.

**Comment by jonathancross** [comment]
FYI: BOLT12 is live in Phoenix
